packages/keystone/lib/Keystone/index.js

```diff
@@ -234,17 +234,16 @@ module.exports = class Keystone {
     return execute(schema, query, null, context, variables);
   }
 
-  // The GraphQL App uses this method to build up the context required for each
-  // incoming query.
-  // It is also used for generating the `keystone.query` method
-  getGraphQlContext({ schemaName, req = {}, skipAccessControl = false } = {}) {
+  createHTTPContext({ schemaName, req }) {
```

Comment:
Do we need `req = {}` here?

Reply:
No, `req` is a required argument for this use case, which is specifically for HTTP requests.